dashboard/test/factories/course_offerings.rb

```diff
@@ -0,0 +1,21 @@
+# == Schema Information
+#
+# Table name: course_offerings
+#
+#  id         :integer          not null, primary key
+#  name       :string(255)
+#  properties :text(65535)
+#  created_at :datetime         not null
+#  updated_at :datetime         not null
+#
+# Indexes
+#
+#  index_course_offerings_on_name  (name) UNIQUE
+#
+
+FactoryGirl.define do
+  factory :course_offering do
+    name "MyString"
+    properties "MyText"
+  end
+end
```

Comment:
if you do add these, I'd recommend following the existing pattern of adding these to factories.rb, unless you feel like reading opinions on the internet about whether we should change our practice here